pixels-turbo/pixels-worker-common/src/main/java/io/pixelsdb/pixels/worker/common/BaseAggregationWorker.java

```diff
@@ -137,6 +137,7 @@ public AggregationOutput process(AggregationInput event)
             WorkerCommon.initStorage(inputStorageInfo);
             WorkerCommon.initStorage(outputStorageInfo);
 
+            logger.info("start get output schema");
```

Comment:
Remove these logs as aggregation is not considered in this pr.

Reply:
Done, but just to explain, `BaseScanWorker` has these info log while `BaseAggregationWorker` does not. Will need to add in the next PRs